src/main/features/core/tray.js

```diff
@@ -219,3 +234,8 @@ Emitter.on('audiooutput:list', (event, devices) => {
 
```

Comment:
I can't actually comment on the line I want to, but line 232

```js
setContextMenu();
```

will cause your logic to break.  You aren't passing in a track so it will reset the string to "Not Playing".  You should pass in `PlaybackAPI.currentSong()` in this call.

**Sidenote:** I need to update `currentSong` to be `currentTrack` 😆